nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -86,6 +86,53 @@
 }</programlisting>
     </para>
    </listitem>
+
+   <listitem>
+     <para>
+       The PHP package is now a much smaller package with defaults that are much closer to upstream
+       defaults for which extensions that should be enabled. Notable extensions that are disabled now
+       that used to be enabled contain among other things database related extensions.
+     </para>
+     <para>
+       Since this release we have an easy way to customize your PHP install to get
+       additional extensions enabled. See following snippet to install a PHP with <literal>gd</literal>,
+       <literal>pdo_mysql</literal> and <literal>pdo_pgsql</literal> added:
+
+       <programlisting>
```

Comment:
Is this properly indented in the rendered manual? IIRC `<programlisting>`-tags are completely unindented in our docs.

Reply:
I wouldn't know, I'm not very familiar with docbook. I was reading previous releasenotes to find good examples. And I think it is based on the firejail entry here: https://nixos.org/nixos/manual/release-notes.html#sec-release-18.09-highlights

Reply:
Also no idea :D

I'd recommend to review it using e.g. `firefox ./result/share/doc/nixos/index.html` after running `nix build --print-build-logs -f nixos/release.nix --arg supportedSystems '[ builtins.currentSystem ]' --option builders '' manual` (I really command an alias for the building stuff at least :p)

Reply:
It does indent, but I've fixed up the indentation to align to the left in the rendered manual :)
![image](https://user-images.githubusercontent.com/461970/76894114-ce5b7880-688d-11ea-9612-2f822e963e70.png)

Reply:
You can just use `nix-build nixos/release.nix -A manualHTML.x86_64-linux`. And `nix-build pkgs/top-level/release.nix -A manual` for nixpkgs.